src/conv.jl

```diff
@@ -86,8 +88,9 @@ for backend in (Symbol(), :_direct, :_im2col, :_nnpack)
             function $(Symbol("$(name)$(backend)"))(
                             x::AbstractArray{xT,N}, w::AbstractArray{wT,N},
                             cdims::ConvDims; kwargs...) where {xT, wT, N}
-                y = similar(x, promote_type(xT, wT), output_size(cdims)...,
-                               channels_out(cdims), size(x,N))
+                y = similar(
+                   x, promote_type(xT, wT), output_size(cdims)...,
+                   channels_out(cdims), size(x,N))
```

Comment:
Best if we can revert the syntax changes that aren't actually code changes. It makes it easier to remain consistent across the package and focus the PRs on the relevant changes.